conceptnet5/builders/combine_assertions.py

```diff
@@ -127,18 +131,6 @@ def group_func(line):
     out_bad.close()
 
 
-def output_assertion(out, **kwargs):
-    """
-    Output an assertion to the given output stream. All keyword arguments
-    become arguments to `make_edge`. (An assertion is a kind of edge.)
-    """
-    # Build the assertion object.
-    assertion = make_edge(**kwargs)
-
-    # Output the result in a Msgpack stream.
-    out.write(assertion)
-
-
 class AssertionCombiner(object):
```

Comment:
This was the case since long before, but I believe the AssertionCombiner is no longer necessary in our current test framework.